tools/pot/openvino/tools/pot/engines/data_free_engine.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2021-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from openvino.tools.pot.configs.config import Config
+from openvino.tools.pot.data_loaders.synthetic_image_loader import SyntheticImageLoader
+from openvino.tools.pot.engines.simplified_engine import SimplifiedEngine
+
+class DataFreeEngine(SimplifiedEngine):
+    def __init__(self, config, data_loader=None, metric=None):
+        super().__init__(config)
+        if not data_loader:
+            self._data_loader = self.get_data_loader(config)
```

Comment:
We need a unit test that will verify e2e workflow: from image generation to quantization using this image. @l-bat, please consider adding it along with this PR.